python/lsst/source/injection/inject_engine.py

```diff
@@ -25,7 +25,8 @@
 
 import os
 from collections import Counter
-from typing import Any, Generator
+from collections.abc import Generator
+from typing import Any
```

Comment:
As below with `collections.abc.Sequence`, is there a good reason to favour one over the other here?

Reply:
Anything that is in `collections.abc` is now always preferred and the `typing` variant is deprecated.

Reply:
(see @timj's reply above, thanks :+1:)